clarity/evaluator/haspi/eb.py

```diff
@@ -876,9 +875,6 @@ def env_compress_basilar_membrane(
     Two-tone suppression added 22 August 2008.
     Translated from MATLAB to Python by Zuzanna Podwinska, March 2022.
     """
-    # Initialize the compression parameters
-    threshold_high = 100
-
```

Comment:
Because I remove this hard-coded value of `threshold_high`, I had to change the test to 100